### PR TITLE
More proofreading of pathogenesis manuscript

### PR DESCRIPTION
**Pull request by ajlee21**
### Description of the proposed additions or changes

This PR has mostly minor text edits after proofreading. Some edits might be stylistic so please feel free to ignore if you disagree.

I think this manuscript sounds great!

A couple of additional items of note:
1. I noticed there is the abbrev ACE and hACE. Do we want to use just one here? I was assuming the ACE references were also in human.
2. I'm not sure if there is an issue with some of the latex symbols in the Transmission section with `R~0~` the number if crossed out. Also `$\gamma$` isn't showing as the greek symbol but as the formula in the text.
3. The manuscript mentioned that damage could be the result of the immune response and the pathogen itself. I mainly thought that symptoms resulted from the immune system fighting off the virus. I'm just curious about the contribution of the pathogen itself. Is this referring to the virus killing the cells that it uses to replicate?

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/07.pathogenesis.md [hunk 1]
```diff
@@ -324,9 +324,9 @@ Viral particles can enter the body if they then come in contact with the oral, n
 Droplet transmission occurs when a contagious individual sneezes, coughs, or exhales and produces respiratory droplets that can contain a large number of viral particles [@pubmed:23771256].
 Contact with these droplets can occur either through direct exposure to the droplets, such as breathing in droplets produced by a sneeze, or exposure to particles that have settled on a surface [@pubmed:23771256].
 Aerosol transmission refers to much smaller particles (less than 5 micrometers) that are also produced by sneezing, coughing, or exhaling [@doi:10.1097/QCO.0000000000000563; @pubmed:23771256].
-The small size of these particles allows them to remain suspended over a longer period of time and potentially to be moved by air currents [@pubmed:23771256].
+The small size of these particles allow them to remain suspended over a longer period of time and potentially to be moved by air currents [@pubmed:23771256].
```
Comment (agitter) [inline content/07.pathogenesis.md:327]: "size... allows" reads better to me here

**Review by agitter — COMMENTED** [review]
Looks good to me except for the one change I commented on.

> I'm not sure if there is an issue with some of the latex symbols in the Transmission section with R~0~ the number if crossed out. Also $\gamma$ isn't showing as the greek symbol but as the formula in the text.

Do you see these formatting errors in the HTML version https://greenelab.github.io/covid19-review/#dynamics-of-transmission?  I didn't notice them.

Let's wait to merge until someone can respond to your comments 1) and 3).

File: content/07.pathogenesis.md [hunk 2]
```diff
@@ -118,7 +118,7 @@ Coronaviruses such as SARS-CoV-1 are also able to evade the humoral immune respo
 
 #### Host Cell Susceptibility
 
-ACE2 and TMPRSS2 have been identified as the primary entry portal and as a critical protease, respectively, in facilitating the entry of SARS-CoV-1 and SARS-CoV-2 into a target cell [@doi:10.1038/nature02145; @doi:10.1126/science.abb2507; @doi:10.1128/JVI.01542-10; @doi:10.1016/j.cell.2020.02.052; @doi:10.1128/JVI.02232-10].
+ACE2 and TMPRSS2 have been identified as the primary entry portals and as a critical proteases, respectively, in facilitating the entry of SARS-CoV-1 and SARS-CoV-2 into a target cell [@doi:10.1038/nature02145; @doi:10.1126/science.abb2507; @doi:10.1128/JVI.01542-10; @doi:10.1016/j.cell.2020.02.052; @doi:10.1128/JVI.02232-10].
```
Comment (rando2) [inline content/07.pathogenesis.md:121]:
```suggestion
ACE2 and TMPRSS2 have been identified as the primary entry portal and as a critical protease, respectively, in facilitating the entry of SARS-CoV-1 and SARS-CoV-2 into a target cell [@doi:10.1038/nature02145; @doi:10.1126/science.abb2507; @doi:10.1128/JVI.01542-10; @doi:10.1016/j.cell.2020.02.052; @doi:10.1128/JVI.02232-10].
```
ACE2 maps to entry portal and TMPRSS2 maps to protease, so singular is correct here!

File: content/07.pathogenesis.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ The current coronavirus disease 2019 (COVID-19) pandemic, caused by the _Severe
 Symptoms of the disease can range from mild to severe to fatal [@url:https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html] and can affect a variety of organs and systems [@doi:10.23750/abm.v91i1.9397].
 Outcomes of the disease can include acute respiratory distress (ARDS) and acute lung injury, as well as damage to other organ systems [@doi:10.23750/abm.v91i1.9397; @doi:10.1002/ctm2.16].
 Understanding the progression of the disease, including these diverse symptoms, depends on understanding how the virus interacts with the host.
-Additionally, the fundamental biology of the virus can provide insights into how it is transmitted among people, which are in turn critical to efforts to control its spread.
+Additionally, the fundamental biology of the virus can provide insights into how it is transmitted among people, which can in turn inform efforts to control its spread.
```
Comment (rando2) [inline content/07.pathogenesis.md:28]:
```suggestion
Additionally, the fundamental biology of the virus can provide insights into how it is transmitted among people, which can, in turn, inform efforts to control its spread.
```

File: content/07.pathogenesis.md [hunk 4]
```diff
@@ -164,14 +164,14 @@ This excessive clotting in lung capillaries has been suggested to be related to
 Symptoms of a disease can be caused by a pathogen, but they can also be caused by the immune system's reaction to the pathogen.
 A dysregulated immune response can cause significant damage to the host [@isbn:9780815332183; @isbn:9780071283663; @doi:10.1016/j.chom.2020.05.009].
 The inflammatory response has received particular attention for its role in both a healthy response to infection and a pathogenic one.
-Inflammation is one of the most visible components of the immune response, as it is responsible for hallmarks of injury such as pain, heat, and swelling [@doi:10.18632/oncotarget.23208].
+Inflammation is one of the most visible components of the immune response, as it is responsible for the hallmarks of injury such as pain, heat, and swelling [@doi:10.18632/oncotarget.23208].
```
Comment (rando2) [inline content/07.pathogenesis.md:167]:
```suggestion
Inflammation is one of the most visible components of the immune response, as it is responsible for the hallmarks of injury, such as pain, heat, and swelling [@doi:10.18632/oncotarget.23208].
```

File: content/07.pathogenesis.md [hunk 5]
```diff
@@ -181,7 +181,7 @@ On a shorter timescale, dysregulated systemic inflammation can cause sepsis, whi
 
 Cytokines have been investigated for their role in the immune response to lung infections long before the COVID-19 pandemic.
 Dysregulation of the inflammatory response, including elevated levels of pro-inflammatory cytokines, is found in patients with ARDS, which is a severe condition that can arise from pneumonia, SARS, and COVID-19 [@doi:10.1159/000087104].
-One study of patients with and at risk for ARDS (specifically, those who were intubated for medical ventilation) found that shortly after the onset of ARDS, anti-inflammatory cytokine concentration in BALF increased relative to the concentration of pro-inflammatory cytokines [@doi:10.1164/ajrccm.164.10.2104013].
+One study of patients with and at risk for ARDS, specifically those who were intubated for medical ventilation found that shortly after the onset of ARDS, anti-inflammatory cytokine concentration in BALF increased relative to the concentration of pro-inflammatory cytokines [@doi:10.1164/ajrccm.164.10.2104013].
```
Comment (rando2) [inline content/07.pathogenesis.md:184]:
```suggestion
One study of patients with and at risk for ARDS, specifically those who were intubated for medical ventilation, found that shortly after the onset of ARDS, anti-inflammatory cytokine concentration in BALF increased relative to the concentration of pro-inflammatory cytokines [@doi:10.1164/ajrccm.164.10.2104013].
```

File: content/07.pathogenesis.md (hunk 1 above)
Comment (rando2) [reply content/07.pathogenesis.md:327]:
```suggestion
The small size of these particles allows them to remain suspended over a longer period of time and potentially to be moved by air currents [@pubmed:23771256].
```
I agree with @agitter, it simplifies to "The size allows them to..."

**Review by rando2 — APPROVED** [review]
Looks good @ajlee21! I just had a couple minor changes.

**Comment by rando2** [comment]
Thank you for the feedback @ajlee21! Here is my understanding of responses:

> 1. I noticed there is the abbrev ACE and hACE. Do we want to use just one here? I was assuming the ACE references were also in human.

I believe the term hACE/hACE2 was originally introduced to the manuscript by @byrdjb (who is one of the MDs on the project). My assumption was that we should use hACE2 when specifically talking about interactions between the human body/immune system and the virus, and use ACE2 when extrapolating more broadly from _in vitro_ or animal model studies. However, I am pretty new to medical research, so I could be wrong about this! This is just what I would do in an evolutionary genetics context.

> 2. I'm not sure if there is an issue with some of the latex symbols in the Transmission section with R~0~ the number if crossed out. Also $\gamma$ isn't showing as the greek symbol but as the formula in the text.

I haven't noticed this being an issue while going through the Word document edits provided by some of the authors, so I believe it is rendering properly in the docx export. I also didn't see any issues in the HTML rendering but would love to know if I'm mistaken! There was a typo in this section where a semicolon got inserted into a protein name where it wasn't needed, and that obviously did not render correctly, but I haven't noticed any issues with the others! For the record, it does seem that just pasting in the Greek symbols directly does work, which might be better moving forward since it will render correctly on the command line and in raw markdown (I was initially worried about this and changed a bunch of them to the md format!!! 🤦‍♀️ )

> 3. The manuscript mentioned that damage could be the result of the immune response and the pathogen itself. I mainly thought that symptoms resulted from the immune system fighting off the virus. I'm just curious about the contribution of the pathogen itself. Is this referring to the virus killing the cells that it uses to replicate?

I think this is generally the case that the "illness" we perceive is caused by the immune system (fever, sneezing, etc.) but with SARS-CoV-2, the virus is [damaging cells (esp lung cells)](https://www.sciencedaily.com/releases/2020/12/201202114502.htm) while the immune system is trying to stop it, so you see the negative effects of both the pathogen and the immune response!

**Comment by ajlee21** [comment]
Thanks for the grammar help!

I'll commit those changes. 

Your responses make sense. I didn't notice an issue with the HTML as you both point out. I guess it was just the github rendering.

**Pull request merged**